content/en/blog/_posts/2022-05-13-grpc-probes-in-beta.md

```diff
@@ -115,7 +115,7 @@ metadata:
 spec:
   containers:
   - name: agnhost
-    image: k8s.gcr.io/e2e-test-images/agnhost:2.35
+    image: registry.k8s.io/e2e-test-images/agnhost:2.35
```

Comment:
```suggestion
    # image changed since publication (previously used registry "k8s.gcr.io")
    image: registry.k8s.io/e2e-test-images/agnhost:2.35
```